components/charts/types/__tests__/LineChartCustomizations.test.tsx

```
@@ -0,0 +1,103 @@
+    expect(screen.getByLabelText('X-Axis Label Rotation')).toBeInTheDocument();
+    expect(screen.getByLabelText('Y-Axis Label Rotation')).toBeInTheDocument();
+
+    const xInput = screen.getByLabelText('X-Axis Title');
+    await user.type(xInput, 'D');
+    expect(mockUpdateCustomization).toHaveBeenCalledWith('xAxisTitle', 'MonthsD');
+  });
+
+  it('should disable all controls when disabled is true', () => {
+    render(<LineChartCustomizations {...defaultProps} disabled={true} />);
+
+    screen.getAllByRole('switch').forEach((s) => expect(s).toBeDisabled());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)